### PR TITLE
Add Finnish localization

### LINKED ISSUE #22: Add Finnish translation
### Checklist

- [X] I read and understood the **[Contributing Guidelines](https://github.com/Abrynos/ShoppingList/blob/main/.github/CONTRIBUTING.md)**.
- [X] This is not a **[question](https://github.com/Abrynos/ShoppingList/discussions)**.
- [X] This is not a **[technical issue](https://github.com/Abrynos/ShoppingList/discussions)**.
- [X] This is not a **[duplicate](https://github.com/Abrynos/ShoppingList/issues)** of an existing issue.
- [X] This idea is not already possible.
- [X] I believe my idea falls into the projects scope and should be offered as part of the built-in functionality.

### Purpose

Adding Finnish translation for the app would allow Finnish people to use it more easily

### Solution

I am able to add the translation

### Why are the currently avialable solutions not sufficient?

Finnish is currently missing

### Can you help us with this feature request?

Yes, I can code the solution myself and send a merge request.

### Additional information

This is the first time I come across a template like this, I hope I did ok!
Happy Hacktoberfest! I was browsing topics under translation for a soft start of the season and this project seemed like a nice one to start with if you are participating this year.

### PR DESCRIPTION
**Pull request by minna-xD**
## Checklist

- [x] I read and understood the **[Contributing Guidelines](https://github.com/Abrynos/ShoppingList/blob/main/.github/CONTRIBUTING.md)**.
- [x] This is not a **[duplicate](https://github.com/Abrynos/ShoppingList/pulls)** of an existing merge request.
- [x] This is not a **[translation update](https://crowdin.com/project/abrynosshoppinglist)**.
- [x] I believe this falls into the scope of the project and should be part of the built-in functionality.
- [x] My code follows the **[code style](https://github.com/Abrynos/ShoppingList/blob/main/.github/CONTRIBUTING.md#code-style)** of this project.
- [x] All new and existing tests pass.  (didn't do any testing)

### New functionality

Added Finnish translations for strings.xml

Resolves #22

### CODE REVIEW

File: app/src/main/res/values-fi/strings.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  ~
+  ~ You should have received a copy of the GNU General Public License along
+  ~ with this program; if not, write to the Free Software Foundation, Inc.,
+  ~ 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+  -->
+
+<resources>
+	<string name="shareHost" translatable="false">a.sl</string>
+	<string name="sharePath" translatable="false">/add</string>
+	<string name="shareScheme" translatable="false">https</string>
```
Comment (Abrynos) [inline app/src/main/res/values-fi/strings.xml:22]: These strings have `translatable` set to `false`. Remove them.

**Review by Abrynos — CHANGES_REQUESTED** [review]

**Comment by Abrynos** [comment]
Please ignore that CI failure. I gotta look at that some time 😓

**Comment by Abrynos** [comment]
If it's not a problem to you, I'd like you to merge the latest changes from the main branch. I've changed a tiny bit in the CI pipeline and would like to try those changes out!

Sidenote: Technically you could even tick the "This is not a translation update." part, because it's not an update, but a new addition 😝

**Review by Abrynos — APPROVED** [review]

**Comment by minna-xD** [comment]
I don't speak that fluent git but I did something to merge things. 😄 Looking better?
Also ticked the final checkbox 😉

**Comment by Abrynos** [comment]
Oh my. you branched off the localization branch (which is an automatic one from the crowdin bot) 😅 And that one was just deleted when I merged those localization updates

**Comment by Abrynos** [comment]
which also closed your PR 😑

**Comment by Abrynos** [comment]
Can you open another one, but this time start off the main branch?

**Pull request closed without merging**